src/util/crypto_hash.h

```diff
@@ -14,6 +14,23 @@ class crypto_hash
 
   bool operator<(const crypto_hash &h2) const;
 
+  std::array<unsigned int, 5> to_array() const
+  {
+    // TODO: CPP+20 will have a to_array function
```

Comment:
C++20 is already released. It *does* have this function. As a free-standing function. Since the `hash` array is already publicly accessible, there is no need for this function in the `crypto_hash` class. Maybe move it to the unit test case?